tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,770 @@
+            silentMode = true;
+        }
+        if (cmdLine.hasOption(SUMMARY_PARAM)) {
+            summaryMode = true;
+            summaryString = new StringBuilder();
+            summaryString.append(String.join("", Collections.nCopies(SUMMARY_WIDTH, "-")));
+            summaryString.append(System.getProperty("line.separator"));
+            summaryString.append("Summary for execution of Elytron-Tool command FileSystemEncrypt");
+            summaryString.append(System.getProperty("line.separator"));
+            summaryString.append(String.join("", Collections.nCopies(SUMMARY_WIDTH, "-")));
+            summaryString.append(System.getProperty("line.separator"));
+        }
+        printDuplicatesWarning(cmdLine);
+
+        String realmNameOption = cmdLine.getOptionValue("r");
+        String inputRealmLocationOption = cmdLine.getOptionValue("i");
+        String outputRealmLocationOption = cmdLine.getOptionValue("o");
+        String credentialStoreOption = cmdLine.getOptionValue("c");
+        String secretKeyAliasOption = cmdLine.getOptionValue("k");
+        String hashCharsetOption = cmdLine.getOptionValue("h");
+        String hashEncodingOption = cmdLine.getOptionValue("e");
+        String levelsOption = cmdLine.getOptionValue("l");
+        String bulkConvert = cmdLine.getOptionValue("b");
+
+        if (bulkConvert == null) {
+            if (realmNameOption == null) {
+                realmNameOption = DEFAULT_FILESYSTEM_REALM_NAME;
+            }
+
+            Descriptor descriptor = new Descriptor();
+            descriptor.setFileSystemRealmName(realmNameOption);
+            descriptor.setOutputRealmLocation(Paths.get(outputRealmLocationOption).toString());
+            descriptor.setInputRealmLocation(Paths.get(inputRealmLocationOption).toString());
+            if (hashCharsetOption == null) {
+                descriptor.setHashCharset(StandardCharsets.UTF_8);
+            } else {
+                try {
+                    descriptor.setHashCharset(Charset.forName(hashCharsetOption));
+                } catch (UnsupportedCharsetException e) {
+//                    Error
+                }
+            }
+            if (hashEncodingOption == null) {
+                descriptor.setHashEncoding(Encoding.BASE64);
+            } else {
+                try {
+                    descriptor.setHashEncoding(Encoding.valueOf(hashEncodingOption.toUpperCase()));
+                } catch (IllegalArgumentException | NullPointerException e) {
+//                    Error
+                }
+            }
+            if (levelsOption == null) {
+                descriptor.setLevels(2);
+            } else {
+                try {
+                    descriptor.setLevels(Integer.parseInt(levelsOption));
+                } catch (NumberFormatException e) {
+//                    Error
+                }
+            }
+                if (credentialStoreOption != null && secretKeyAliasOption != null) {
```

Comment:
Looks like there's extra spaces at the beginning of the if here.

Reply:
Fixed